source/text/vss-characters-latin.ads

```diff
@@ -13,7 +13,32 @@ package VSS.Characters.Latin is
    Nul       : constant Virtual_Character := Virtual_Character'Val (16#0000#);
 
    Backspace : constant Virtual_Character := Virtual_Character'Val (16#0008#);
-
+   Character_Tabulation : constant Virtual_Character :=
+     Virtual_Character'Val (16#0009#);
    Line_Feed : constant Virtual_Character := Virtual_Character'Val (16#000A#);
 
+   Form_Feed : constant Virtual_Character := Virtual_Character'Val (16#000C#);
+   Carriage_Return : constant Virtual_Character :=
+     Virtual_Character'Val (16#000D#);
+
+   Space     : constant Virtual_Character := Virtual_Character'Val (16#0020#);
```

Comment:
Why not use character literals after `:=`?

Reply:
To avoid any visual confusions.